Source/WebGPU/WebGPU/Texture.mm

```diff
@@ -2031,17 +2031,33 @@ static MTLStorageMode storageMode(bool deviceHasUnifiedMemory, bool supportsNonP
 #endif
 }
 
-Ref<Texture> Device::createTexture(const WGPUTextureDescriptor& descriptor, IOSurfaceRef ioSurfaceBacking)
+Ref<Texture> Device::createTexture(const WGPUTextureDescriptor& descriptor)
 {
     Vector<WGPUTextureFormat> viewFormats;
-    if (descriptor.nextInChain) {
-        if (descriptor.nextInChain->sType != static_cast<WGPUSType>(WGPUSTypeExtended_TextureDescriptorViewFormats))
-            return Texture::createInvalid(*this);
-        const auto& descriptorViewFormats = reinterpret_cast<const WGPUTextureDescriptorViewFormats&>(*descriptor.nextInChain);
-        if (descriptor.nextInChain->next != nullptr)
+    IOSurfaceRef ioSurfaceBacking = nullptr;
+    const auto* current = descriptor.nextInChain;
+    while (current) {
+        bool viewFormatsSpecified = false;
+        bool ioSurfaceBackingSpecified = false;
```

Comment:
`auto`

Reply:
How is auto better than bool? Same length, less information.

Reply:
Oh interesting, I checked https://webkit.org/code-style-guidelines/ and indeed `auto` isn't a strict rule 👍

Reply:
Right, we do prefer `auto` in general but for longer types. For things like `int` or `bool`, we don't usually use `auto` AFAIK.